src/libs/ReceiptUtils.ts

```diff
@@ -33,7 +33,7 @@ type FileNameAndExtension = {
  * @param receiptPath
  * @param receiptFileName
  */
-function getThumbnailAndImageURIs(transaction: OnyxEntry<Transaction>, receiptPath: string | null = null, receiptFileName: string | null = null): ThumbnailAndImageURI {
+function getThumbnailAndImageURIs(transaction: OnyxEntry<Transaction> | undefined, receiptPath: string | null = null, receiptFileName: string | null = null): ThumbnailAndImageURI {
```

Comment:
I think you can pass `null` instead of `undefined` to keep typing easier. To simplify you can add `null` as a default value in `MoneyRequestConfirmationList`. What do you think?
```suggestion
function getThumbnailAndImageURIs(transaction: OnyxEntry<Transaction>, receiptPath: string | null = null, receiptFileName: string | null = null): ThumbnailAndImageURI {
```

Reply:
At the beginning I did that but then I realised I need to do it in a lot of places and decided that adding `undefined` to this type will be more readable